src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -38,47 +46,101 @@
     public Object[][] gvcfsToGenotype() {
         String basePairGVCF = "gvcf.basepairResolution.gvcf";
         return new Object[][]{
-                {basePairGVCF, "gvcf.basepairResolution.output.vcf", STAND_CALL_CONF_30}, //base pair level gvcf
-                {"testUpdatePGT.gvcf", "testUpdatePGT.output.vcf", STAND_CALL_CONF_30},   //testUpdatePGTStrandAlleleCountsBySample
-                {"gvcfExample1.vcf", "gvcfExample1.vcf.expected.vcf", STAND_CALL_CONF_30}, //single sample vcf
-                {"combined_genotype_gvcf_exception.vcf", "combined_genotype_gvcf_exception.output.vcf", STAND_CALL_CONF_30}, //test that an input vcf with 0/0 already in GT field is overwritten
-                {"combined_genotype_gvcf_exception.nocall.vcf", "combined_genotype_gvcf_exception.output.vcf", STAND_CALL_CONF_30},  //same test as above but with ./.
-                {basePairGVCF, "ndaTest.expected.vcf", Arrays.asList("-nda", "-stand_call_conf", "30")},  //annotating with the number of alleles discovered option
-                {basePairGVCF, "maxAltAllelesTest.expected.vcf", Arrays.asList("--maxAltAlleles", "1", "-stand_call_conf", "30")}, //restricting the max number of alt alleles
-                {basePairGVCF, "standardConfTest.expected.vcf", Arrays.asList("-stand_call_conf", "300")}, //changing call confidence
-                {"spanningDel.combined.g.vcf", "spanningDel.combined.g.vcf.expected.vcf", STAND_CALL_CONF_30},
-                {"spanningDel.delOnly.g.vcf", "spanningDel.delOnly.g.vcf.expected.vcf", STAND_CALL_CONF_30},
-                {"CEUTrio.20.21.gatk3.4.g.vcf", "CEUTrio.20.21.expected.vcf", Arrays.asList("--dbsnp", "src/test/resources/large/dbsnp_138.b37.20.21.vcf", "-stand_call_conf", "30")},
-                //{basePairGVCF, "gvcf.basepairResolution.includeNonVariantSites.expected.vcf", Collections.singletonList("--includeNonVariantSites")
+//                {"combine.single.sample.pipeline.1.vcf", null, Arrays.asList("-V", getTestFile("combine.single.sample.pipeline.2.vcf").toString() , "-V", getTestFile("combine.single.sample.pipeline.3.vcf").toString()), b37_reference_20_21}, //combine not supported yet
```

Comment:
opened to be more explicit about it https://github.com/broadinstitute/gatk/issues/2584